src/dali/dali_main.py

```diff
@@ -63,6 +63,7 @@ class _InputPluginHelperArgs(NamedTuple):
 
 
 def dali_main(country: AbstractCountry) -> None:
+    DEFAULT_CONFIGURATION[Keyword.COUNTRY.value] = country
```

Comment:
Ok I updated. I am still not sure I am understanding what your meaning by "global". I didn't think you meant just stuffing the python globals, cause I didn't seem to see this being done anywhere else in the code. So I delcared COUNTRY in the scope of the configuration module, and then `configuration.COUNTRY` is patched directly in `dali_main()` so it can be accessed globally but is still namespaced under the dali configuration.

I am probably misunderstanding what you are wanting